main.cpp

```diff
@@ -52,8 +56,9 @@ int main(int argc, char *argv[]) {
         delete server;
     } else if (mode == Conts::JASMINEGRAPH_RUNTIME_PROFILE_WORKER){
         if (argc < 4) {
-            std::cout << "Need three arguments. Use 2 <serverPort> <serverDataPort> to "
-                      << "start as worker ." << std::endl;
+//            std::cout << "Need three arguments. Use 2 <serverPort> <serverDataPort> to "
```

Comment:
Lets remove these commented lines. Please do the same with the other similar commented lines in this PR.